.github/workflows/linux-build.yml

```
@@ -50,9 +49,171 @@ permissions:
+          source scripts/setup-ubuntu.sh && install_apt_deps
+
+      # - name: Install Docs Dependencies
+      #   run: |
+      #     dnf install -y --setopt=install_weak_deps=False pandoc
+      #     pip3 install pipx
+      #     pipx install --force uv cmake==3.31
+      #     echo "PATH=$PATH:/github/home/.local/bin" >> $GITHUB_ENV
+      #     which uv
+      #     make python-venv
+      #     which uv
+      #     uv pip install -r scripts/docs-requirements.txt 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)